docs/pages/eas-update/migrate-to-eas-update.md

```diff
@@ -50,25 +48,25 @@ You'll need to make the following changes to your project:
 
 5. To allow updates to apply to builds built with EAS, update your EAS build profiles in **eas.json** to include `channel` properties. These channels should replace any `releaseChannel` properties. We find it convenient to name the `channel` after the profile's name. For instance, the `preview` profile has a `channel` named `"preview"` and the `production` profile has a `channel` named `"production"`.
 
-  ```json
-  {
-    "build": {
-      "development": {
-        "developmentClient": true,
-        "distribution": "internal"
-      },
-      "preview": {
-        "distribution": "internal",
-        "channel": "preview"
-      },
-      "production": {
-        "channel": "production"
-      }
+```json
+{
+  "build": {
+    "development": {
+      "developmentClient": true,
+      "distribution": "internal"
+    },
+    "preview": {
+      "distribution": "internal",
+      "channel": "preview"
+    },
+    "production": {
+      "channel": "production"
     }
   }
-  ```
+}
+```
```

Comment:
The indent in here was to ensure that `CodeBlock` is a part of fifth step list item, instead of being an sole element divining the list entries (same can be seen with `Terminal` components above):

<img width="200" alt="Screenshot 2022-07-15 193317" src="https://user-images.githubusercontent.com/719641/179278159-feed1fbe-4eb7-41a9-8b31-1bfa9a5dc660.png">